libs/sdk-ui-tests-e2e/cypress/integration/03-sdk-ui-charts/pivotTableTransposed.spec.ts

```diff
@@ -0,0 +1,74 @@
+
+describe("Table Component", { tags: ["checklist_integrated_tiger"] }, () => {
+    const table = new Table(".s-table-component-transpose");
+
+    it("should display Metric in row, Column header on top", () => {
+        Navigation.visit("visualizations/pivot-table/pivot-table-transposed-has-mr-row-left");
+        table.waitLoaded().hasMetricHeaderInRow(1, 1, "Amount").hasMetricHeaderInRow(2, 1, "Amount");
+    });
+
+    it("should display successful with Metric in row", () => {
+        Navigation.visit("visualizations/pivot-table/pivot-table-transposed-has-mr-row-top");
+        table.waitLoaded().hasMetricHeaderInRow(1, 1, "Amount").hasMetricHeaderInRow(2, 1, "Amount");
+    });
+
+    it("should show Row attribute and Column header on top", () => {
+        Navigation.visit("visualizations/pivot-table/pivot-table-transposed-has-rc-row-left");
+        table.waitLoaded().hasHeader("Product").hasColumnHeaderOnTop("Forecast Category");
+    });
+
+    it("should show Row header on top", () => {
+        Navigation.visit("visualizations/pivot-table/pivot-table-transposed-has-r-row-left");
+        table.waitLoaded().hasHeader("Product");
+    });
+
+    it("should show Metric in row", () => {
+        Navigation.visit("visualizations/pivot-table/pivot-table-transposed-has-m-row-left");
+        table.waitLoaded().hasMetricHeaderInRow(0, 0, "Amount");
+    });
+
+    it("should show Column header on top", () => {
+        Navigation.visit("visualizations/pivot-table/pivot-table-transposed-has-c-left");
+        table.waitLoaded().hasColumnHeaderOnTop("Forecast Category");
+    });
+});
+
+describe("Insight View", { tags: ["checklist_integrated_tiger"] }, () => {
+    const table = new Table(".s-insight-view-transpose");
+
+    it("should display Metric in row, Column header on left", () => {
+        Navigation.visit("insight/insight-transpose-has-mc-row-left");
+        table.waitLoaded().hasCellValue(0, 0, "Forecast Category").hasMetricHeaderInRow(1, 0, "Amount");
+    });
+
+    it("should display Metric in column, Column header on top", () => {
+        Navigation.visit("insight/insight-transpose-has-mc-column-top");
+        table
+            .waitLoaded()
+            .hasColumnHeaderOnTop("Forecast Category")
+            .hasMeasureHeader(1, "Amount", true)
+            .hasMeasureHeader(2, "Amount", true);
+    });
+
+    it("should display Metric in column, Column header on the top", () => {
+        Navigation.visit("insight/insight-transpose-has-mc-column-on-top");
+        table
+            .waitLoaded()
+            .hasColumnHeaderOnTop("Forecast Category")
+            .hasMeasureHeader(0, "Amount", true)
+            .hasMeasureHeader(1, "Amount", true);
+    });
+
+    it("should display Metric in row", () => {
+        Navigation.visit("insight/insight-transpose-has-mc-row");
+        table.waitLoaded().hasMetricHeaderInRow(0, 0, "Amount");
+    });
+
+    it("should display Column header on top", () => {
```

Comment:
Wrong case in E2E, this case should config with 
`config={     
            columnHeadersPosition:”left”
 }`
Please help me to update it, many thanks!